packages/circuit-ui/components/Button/Button.tsx

```diff
@@ -196,55 +196,55 @@ const primaryStyles = ({
   if (destructive) {
     return css`
       background-color: var(--cui-bg-danger-strong);
-      border-color: var(--cui-border-danger);
+      border-color: transparent;
```

Comment:
I didn't know that a `transparent` border would effectively result in a border of the background's color here (since it's transparent but wtill with a 1px width)! Thought the border area was an extra px outline _around_ the element, but it looks like it's an outline _over_ the element (over its outermost pixel-wide outline). TIL

Reply:
I had the same revelation, and then realized that this is determined by the [CSS box model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model) 💡

Reply:
❤️ CSS